applications/msai.md

```diff
@@ -0,0 +1,290 @@
+Essentially we would like to implement something similar to w3c Solid pod but integrated with a blockchain node Substrate node that can be used by all healthcare applications.
+Integrating application specific security and storage for every program is a major pain point which is not addressed by blockchains.
+A combination of a Substrate blockchain, because of its modularity and Solid pod like api [5] would provide the security, decentralisation and immutable record keeping that is needed to make healthcare interoperable.
+To make this feasible it needs to be split into stages initially using a smart contract developed in Ink! - WebAssembly which records ownership and access [1] to a data pod with a similar API to Solid PODs [4].
+
+ID verification and data enhancement could be integrated through a Sidecar and or Pallet approach allowing other app developers within the ecosystem to reuse the chain without needing their own backend. 
+For a simpler initial design we would enhance the off chain worker storage api[6]. Distributed data could be verified as authentic by having a secure hash stored on the chain [4].
+With trust being established by the blockchain layer, any blockchain node that provides faulty data as verified through smart contract hashes will be kicked out and forfiet their staking in the parachains native token.
+We also propose specific data format for anonymous tags such that no user identified private data would be exposed
+during the whole process. 
+The initial user of this parachain would be our own EMR application where users store any medical documents with automated information extraction, along with authenticated GPDR opt out.
+Sample screen of users view of how the data will be presented in our implementation can be seen[here](https://docs.google.com/presentation/d/e/2PACX-1vR5uQkX9I_UdCQwDQpNUkyaIQKtYWfY0_ZhDLQHSxdeybF8QqbaYKNQ6j0obcg5BRhEjSmgI3NkhnhP/pub?start=false&loop=false&delayms=3000&slide=id.p1). 
+
+Such a platform would allow game theory incentives and smart contract based marketplaces for data trading to encourage adoption of the reference application. 
+
+Our team has years of research and development on medical data along with senior blockchain developers working with IT capable PhD accredited medical doctor(s). Data collection extraction and real time scaling is also part of Dr S Marvasti's and Dr Mahdi Ghandi's resume .
+Furthermore, Dr Marvasti has multiple years experience working with blockchain technologies including ZKP both for cross border fintech and for data analysis through AWS Neptune and AWS Aurora API based databases.
+This is a phased plan with part of the design to be implemented in this first phase, which each phase being usable .
+in summary benefits of blockchain for this data includes
+* Decentralised: Data ownership is not lost to any central authority and access is controlled through secure  NFT like smart contract
```

Comment:
(note this section addressing general question of the ultimate benefits of blockchain not necessarily version 1)
 1- It can be via various mechanisms which dont require node based storage but simpler when the data is controlled by the nodes. 
2- with etheruem for isntance a proxy service could be created but with SUbstrate we would want the proxy service to be built into the node or as a side car. 
https://en.wikipedia.org/wiki/Proxy_re-encryption or https://www.researchgate.net/publication/327034734_Blockchain-Based_Decentralized_Access_Control_for_IPFS#:~:text=The%20smart%20contract%20is%20used%20to%20maintain%20the,the%20access%20controlled%20IPFS%20is%20analyzed%20and%20discussed..  The smart contract will hold an access control list and the data will be hashed based on account hashes and only hashes which are allowed access will be served. If this was already possible then there would be no need for a separate chain that enforces this. The complexity of initial version will be limited. This is the ultimate intention